gammapy/image/maps.py

```diff
@@ -82,14 +82,28 @@ def read(cls, fobj, *args, **kwargs):
                 fobj = str(make_path(fobj))
             data = fits.getdata(fobj, *args, **kwargs)
             header = fits.getheader(fobj, *args, **kwargs)
-        wcs = WCS(header)
-        meta = header
 
+        image_hdu = fits.ImageHDU(data, header)
+        return cls.from_image_hdu(image_hdu)
+
+    @classmethod
+    def from_image_hdu(cls, image_hdu):
```

Comment:
The `from_image_hdu` docstring has two little formatting issues:
* There should be an empty line before the one with `Parameters`
* There should be a space character before the colon, i.e. `image_hdu :` instead of `image_hdu:`

I know it's painfully slow (a minute or two), but you should still always check your docstrings locally via
```
python setup.py build_sphinx
open docs/_build/html/index.html
```
and then navigate to the docstring you wrote and check that it's formatted correctly.